maistra/local/bin/node

```diff
@@ -0,0 +1 @@
+/bin/node
```

Comment:
Do you need these slinks to be committed? Aren't they created at runtime in the beginning of the build?

Reply:
Yes, it's not necessary to commit them. In fact they are removed and re-created during each build.

Reply:
drive-by comment: probably also best to add them to `.gitignore`?

Reply:
It looks a good idea